docs/Logging.md

```diff
@@ -21,9 +21,12 @@ The library will generate two logs for each request, representing the incoming r
 To enable this logging functionality you should add the following line in the `Configure` function of `Startup.cs` file of your microservice:
 
 ```csharp
-app.UseRequestResponseLoggingMiddleware();
+app.UseRequestResponseLoggingMiddleware(new MiddlewareOptions {excludedPrefixes = new List<string>() { "/-/" }});
```

Comment:
I'd provide the example in different lines so that is more readable

Reply:
I have modified the documentation accordingly! Let me know if it is ok now!